### PR TITLE
triton#292 Equinix Metal cli syntax change

### LINKED ISSUE #292: Equinix Metal cli syntax change
Equinix has moved over from `packet-cli` to `metal-cli` and changed the syntax in some cases.

### PR DESCRIPTION
**Pull request by bahamat**
Changes made during set up of a new demo environment.

Fixes #292

### CODE REVIEW

**Comment by danmcd** [comment]
What is it with changing interfaces for no seemingly good reason?

File: tools/triton-eqm-create.sh [hunk 1]
```diff
@@ -71,8 +74,8 @@ function usage
     printf 'Create a new Triton Headnode install in Equinix Metal.\n\n'
     printf 'Syntax:\n\n'
     printf '\t%s project -n project_name\n' "$0"
-    printf '\t%s headnode -p project_uuid -f facility -P hardware_plan -a answer_file\n\n' "$0"
-    printf '\t%s computenode -n <cn name> -p project_uuid -f facility\n\n' "$0"
+    printf '\t%s headnode -p project_uuid -f facility -f facility -P hardware_plan -a answer_file\n\n' "$0"
```
Comment (danmcd) [inline tools/triton-eqm-create.sh:77]: Why two "-f facility" options?

Comment (bahamat) [reply tools/triton-eqm-create.sh:77]: A mistake :-)

Comment (bahamat) [reply tools/triton-eqm-create.sh]: Fixed.

**Review by danmcd — APPROVED** [review]

**Comment by bahamat** [comment]
If there's any good time, this was it. They changed the name of the binary to reflect the acquisition. The old binary worked fine, but was no longer available for new users to download. So the syntax changes going along with a new binary name makes sense.

**Pull request merged**